content/dota_addons/legion_td/panorama/layout/custom_game/main_hud.xml

```diff
@@ -11,8 +11,8 @@
     <scripts>
         <include src="file://{resources}/scripts/custom_game/utils.js" />
         <include src="file://{resources}/scripts/custom_game/main_hud.js" />
-        <include src="file://{resources}/scripts/custom_game/legion_creator_panel.js" />
         <include src="file://{resources}/scripts/custom_game/health_bars.js" />
+        <include src="file://{resources}/scripts/custom_game/team_select_player.js" />
```

Comment:
This does not really make sense. The `team_select` scripts should solely contain logic for the team selection on the loading screen. Please move the logic you wanna access on the main hud to another script (like the `legion_creator_panel.js`).